nextcord/guild.py

```diff
@@ -796,6 +802,19 @@ def public_updates_channel(self) -> Optional[TextChannel]:
         channel_id = self._public_updates_channel_id
         return channel_id and self._channels.get(channel_id)  # type: ignore
 
+    @property
+    def safety_alerts_channel(self) -> Optional[TextChannel]:
+        """Optional[:class:`TextChannel`]: Returns the guild's safety alerts channel where admins and
+        moderators of the guild receive notices about DM or join activity that exceeds usual numbers
+        for the guild.
```

Comment:
```suggestion
        """Optional[:class:`TextChannel`]: Returns the guild's channel where admins and
        moderators of the guild receive safety alerts from Discord. The guild must be a 
        Community guild.
```
This is a little more broad, which is good if and when the safety alerts channel gets different alerts in the future.